src/Elliptic/Systems/IrrotationalBns/FirstOrderSystem.hpp

```diff
@@ -28,33 +25,39 @@ namespace IrrotationalBns {
  *  formulated as a set of coupled first-order PDEs.
  *
  * \details This system formulates the Irrotational Bns HSE equations for the
- *  velocity potential \f$\Phi\f$. For a background matter distribution (given
- *  by the specific enthalpy h) and a background metric \f$\gamma_{ij}\f$.
- *
- * \f{align*}
- * D_i U^i = -U^i D_i \ln \left(\frac{\alpha}{h}) \\
-
- * \f}
- *
- * where we have chosen the velocity potential gradient as an auxiliary
- * variable \f$D_i \Phi = U_i \equiv h u_i\f$ with \f$u_i\f$ the four velocity
+ * velocity potential \f$\Phi\f$. For a background matter distribution (given
+ * by the specific enthalpy h) and a background metric \f$\gamma_{ij}\f$.
+ * The velocity potential is defined by \f$D_i \Phi = h u_i\f$ with \f$u_i\f$
+ (the spatial part of) the four velocity
  * and where \f$\Gamma^i_{jk}=\frac{1}{2}\gamma^{il}\left(\partial_j\gamma_{kl}
  * +\partial_k\gamma_{jl}-\partial_l\gamma_{jk}\right)\f$ are the Christoffel
- * symbols of the second kind of the background metric \f$\gamma_{ij}\f$. The
+ * symbols of the second kind of the background (spatial) metric
+ \f$\gamma_{ij}\f$. The
  * background metric \f$\gamma_{ij}\f$ and the Christoffel symbols derived from
- * it are assumed to be independent of the variables \f$u\f$ and \f$v_i\f$, i.e.
- * constant throughout an iterative elliptic solve.
+ * it are assumed to be independent of the variables \f$\Phi\f$ and \f$u_i\f$,
+ i.e.
+ * constant throughout an iterative elliptic solve.  Additionally a background
+ lapse (\f$\alpha\f$) and
+ * shift (\f$\beta\f$) must be provided.  Finally, a ``rotational killing
+ vector" \f$k^i\f$ (with magnitude
+ * proportional to the angular velocity of the orbital motion) is provided.  The
+ * rotational shift is defined as \f$B^i = \beta^i + k^i\f$ which is
+ heuristically the background
+ * motion of the spacetime.
  *
  * The system can be formulated in terms of these fluxes and sources (see
  * `elliptic::protocols::FirstOrderSystem`):
  *
+ *
+ * \f{align*}
+ * D_i F^i + S = f
```

Comment:
`-\partial_i F^i + S = f` (note the minus sign). This means some minus signs in the equations below need to be fixed too.